shared/markdown/parser.pegjs

```diff
@@ -117,14 +117,20 @@ __INLINE_RULE__<StrikeInline, !StrikeMarker>
 Strike
  = StrikeMarker !WhiteSpace children:StrikeInline StrikeMarker !(StrikeMarker / NormalChar) { return {type: 'strike', children: flatten(children)} }
 
-// children grammar adapted from username regexp in libkb/checkers.go.
+// children test adapted from CheckUsername in libkb/checkers.go.
 Mention
- = MentionMarker children:([a-zA-Z0-9][a-zA-Z0-9_]*) MentionMarker service:ValidMentionService { return {type: 'mention', children: flatten(children), service: service.toLowerCase()} }
+ = MentionMarker children:([a-zA-Z0-9]+"_"?)+ MentionMarker service:ValidMentionService & {
```

Comment:
Hmm, I'm curious as to why `@a_b_c` doesn't get parsed correctly as a mention? 
<img width="125" alt="screen shot 2018-01-08 at 10 56 12 am" src="https://user-images.githubusercontent.com/11968340/34678969-94f515e8-f462-11e7-8043-1b94bf918d29.png">
is the rule to disallow more than one underscore, or just consecutive underscores?

Reply:
Note that a user has to exist for it to be highlighted. There doesn't seem to be an `a_b_c` username. I'll add that to the parser tests, though.

Reply:
Added, PTAL.